tests/integration/models/applicationInvite.spec.ts

```diff
@@ -0,0 +1,130 @@
+// tslint:disable-next-line:import-blacklist
+import * as m from 'mochainon';
+import {
+	balena,
+	BalenaSdk,
+	givenAnApplication,
+	givenLoggedInUser,
+} from '../setup';
+const { expect } = m.chai;
+
+const TEST_EMAIL = 'user.test@example.org';
+const TEST_MESSAGE = 'Hey!, Join my app on balenaCloud';
+const TEST_ROLE = 'developer';
+
+describe('Application Invite Model', function() {
+	givenLoggedInUser(beforeEach);
+	givenAnApplication(beforeEach);
+
+	describe('given no application invite', function() {
+		describe('balena.models.applicationInvite.getAllByApplication()', function() {
+			it('shoud return an empty Array', function() {
+				const promise = balena.models.applicationInvite.getAllByApplication(1);
+				return expect(promise).to.become([]);
+			});
+
+			it('should support a callback with no options', function(done) {
+				// tslint:disable-next-line:ban-types
+				(balena.models.applicationInvite.getAllByApplication as Function)(
```

Comment:
Optional-NIT: Doing the following should allow you to remove the tslint hint.
```suggestion
				(balena.models.applicationInvite.getAllByApplication as (...args: any[]) => any))(
```